tools/Data_explo/11_gala_autocorr.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="tool_autocorr" name="Auto-correlation" version="0.0">
+    <description>Show if there is auto-correlation</description>
+    <macros>
+        <import>macro.xml</import>
+    </macros>
+    <expand macro="Explo_requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+            '$__tool_directory__/Exe11_autocorr.r'
+            '$input'
+            '$variable'
+            '$separator'
+            '$decimal'
+            '$colnames'
+            '$output'
+        ]]>
+        </command>
+        <inputs>
+            <param name="input" type="data" format="txt" label="Input table"/>
+            <param name="variable" label="Select column containing numerical values " type="data_column" data_ref="input" numerical="True" multiple="False"/>
+            <param name="separator" type="text" label="What character is he separator in your dataframe?"/>

```

Comment:
If we agree that all files just take tabular this param could be removed. Galaxy has tools to interconvert tabular files of all sorts so that your tool does not need to deal with that.